src/twisted/test/test_error.py

```diff
@@ -216,28 +259,28 @@ def assertCorrectException(self, errno, message, result, expectedClass):
         self.assertEqual(result.osError, errno)
         self.assertEqual(result.args, (message,))
 
-    def test_errno(self):
+    def test_errno(self) -> None:
         """
         L{error.getConnectError} converts based on errno for C{socket.error}.
         """
         self.assertErrnoException(errno.ENETUNREACH, error.NoRouteError)
         self.assertErrnoException(errno.ECONNREFUSED, error.ConnectionRefusedError)
         self.assertErrnoException(errno.ETIMEDOUT, error.TCPTimedOutError)
-        if platformType == "win32":
+        if sys.platform == "win32":
```

Comment:
Maybe we need type definition for platformType ?

Or if `t.python.runntime.Platform` is no longer relavant , maybe we can deprecate it and just use sys.platform.

Reply:
String comparisons for platform detection are an anti-pattern.  `twisted.python.runtime.platform.isWindows()` is likely preferable here.